gbpservice/neutron/nsf/proxy/cfg.py

```diff
@@ -0,0 +1,5 @@
+OPTS = [
+    {'thread_pool_size': 10},
+    {'unix_bind_path': '/tmp/uds_socket'},
+    {'rest_server_address': '0.0.0.0'},
```

Comment:
Is this the address of configurator? Change rest_server_address -> configurator_address ?